mathext/ell_carlson_test.go

```diff
@@ -0,0 +1,61 @@
+// Copyright ©2017 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package mathext
+
+import (
+	"math"
+	"math/rand"
+	"testing"
+)
+
+// Testing EllipticF (and EllipticRF) using the addition theorems from http://dlmf.nist.gov/19.11.i
+func TestEllipticF(t *testing.T) {
```

Comment:
Is there some kind of absolute values we can compare to? These tests would also pass if `EllipticF` returns 0 for all inputs.